examples/windows/qt-window-app/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.17)
+
+# project(qt_window_app VERSION 0.1 LANGUAGES CXX)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+    add_subdirectory(crashpad)
+endif ()
+
+find_package(Qt5 REQUIRED COMPONENTS Widgets)
```

Comment:
Does this execute correctly on the GitHub pipeline? I'm not seeing the Qt executable (not sure if that is how it works).

Also, should the archive script be updated to archive the demo apps too if they're built?

Reply:
I don't think we want examples in the archive, at least not in the one we currently have (which is used to disseminate pre-built libraries).

As for executing correctly in our CI pipeline: no, it does not, because we don't have Qt 5 available in there. But `examples/CMakeLists.txt` has only builds it if it's available now.

Reply:
Could we apt-get the qt5 stuff?

Reply:
We can install `qt5-default` or `qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools` (as per https://askubuntu.com/a/1335187)